config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/FlamingIdiots-AAAAAAAAAAAAAAAAAAAHNQ==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Blazes are all infernals, and all nasty. Avoid like the plague. Get awesome weapons, awesome armor, and prepare for a fight. \n\nAlternatively, figure out a way to destroy them with technology.\n\nAlternative to the alternative, figure out another way to get blazepowder and blaze rods."
+      "desc:8": "Blazes are all infernals, and all nasty. Avoid like a plague. Get awesome weapons, awesome armor, and prepare for a fight. \n\nThe alternative is to figure out a way to destroy them with technology.\n\nThe alternative to alternative is figure out a different way to get blazepowder and blaze rods."
```

Comment:
should be "The alternative to **the** alternative is **to** figure out"

Reply:
Thank you! Fixed.